src/app/main/component/ubs/components/ubs-order-details/ubs-order-details.component.ts

```diff
@@ -85,6 +85,7 @@ export class UBSOrderDetailsComponent extends FormBaseComponent implements OnIni
 
   ngOnInit(): void {
     this.takeOrderData();
+    this.localStorageService.languageSubject.pipe(takeUntil(this.destroy)).subscribe((lang) => (this.currentLanguage = lang));
```

Comment:
We should also create a getter, so we may accept an array of bags accordingly to current language at any time